schema/test.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "file:///C:/Users/KANOMA1/Documents/projects/release-it/schema/release.json",
```

Comment:
Maybe this file should be omitted entirely?

Reply:
removed in last commit